styleguide/source/assets/css/scss/generic/_mixins.scss

```diff
@@ -26,6 +26,11 @@
   border-top: $width $color $style;
 }
 
+// Horizontal rule
+@mixin rule-horizontal-bottom($width, $color, $style) {
```

Comment:
My thought was that we would probably not be putting`<hr />` tags in the D8 templates, and the rule could be created via the CSS being imported.